docs/querying/sql-window-functions.md

```diff
@@ -88,40 +88,171 @@ You can use the OVER clause to treat other Druid aggregation functions as window
 
 Window functions support aliasing.
 
-## Define a window with the OVER clause
+## Window function syntax
+
+In general, Druid Window functions use the following syntax:
+
+```sql
+SELECT
+    dimensions,
+    aggregation function(s)
+    window_function()
+      OVER ( PARTITION BY partitioning expression
```

Comment:
[PARTITION BY ...] [ORDER BY ...] as these can be optional

Reply:
I think the section about window frames could be moved up here; as that gives a better understanding of partition by and other parts of the frame.

As this sections is about `syntax` - possibly remove the `select` and other non-related things; and only keep
```
window_function() OVER  window
```

I think it would be important to somehow show that the `window` can be specified later as well
```
window_function() OVER  w
from t
WINDOW w AS (PARTITION BY ... )
```